Brewfile

```diff
@@ -0,0 +1,7 @@
+if OS.mac?
+  # We need a version of a JDK older than what's kept in the main cask repository.
+  tap "homebrew/cask-versions"
+  # The JDK formerly known as AdoptOpenJDK, a build of OpenJDK
+  # Use JDK8 because that's a solid base for older Hadoop clusters
```

Comment:
I believe scala 2.11.X / spark 2.3.X is the culprit here w/ requiring java 8.

Reply:
```suggestion
  # Use JDK8 because that's a solid base for older Hadoop clusters
  # Also, Spark 2.3.x essentially requires Scala 2.11.x, and that combination may necessitate Java 8.
```